cfg/common.go

```diff
@@ -171,8 +171,13 @@ type Common struct {
 	Traces               Traces `yaml:"traces"`
 	PrintErrorStackTrace bool   `yaml:"printErrorStackTrace"`
 
-	RenderMismatchCheck             bool `yaml:"renderMismatchCheck"`
-	RenderMismatchMetricReportLimit int  `yaml:"renderMismatchMetricReportLimit"`
+	// RenderReplicaMatchMode indicates how carbonzipper merges the metrics from replica backends.
+	// Possible values are `normal`(default), `check`, and `majority`
+	// `normal` ignores the mismatches and only heals null points.
```

Comment:
The could be indented to mimick bullet points ([not properly support yet](https://github.com/golang/go/issues/7873)).
```// RenderReplicaMatchMode indicates how carbonzipper merges the metrics from replica backends.
	// Possible values are:
	//
	// * `normal` - ignore the mismatches and only heals null points (default)
	//
	// * `check` - look for mismatches, and exposes metrics
	//
	// * `majority` - choose the values of majority of backends in addition to exposing metrics
```